modules/persistent-storage-csi-cloning-overview.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-cloning.adoc
+
+[id="persistent-storage-csi-cloning-overview_{context}"]
+= Overview of CSI volume cloning
+
+A clone represents a duplicate of an existing persistent volume that is its own unique volume on the system at a particular point in time. With cloning, the backend device creates an exact duplicate of the specified volume, rather than creating a new empty volume. Once dynamically provisioned, a clone can be consumed as any standard volume would be.
```

Comment:
Maybe call it a "CSI volume clone" in this first instance for the module?

Reply:
Prefer "After" over "Once" per IBM SG.

Reply:
s/backend/back-end/ since it's an adjective here (IBM SG).

Reply:
> Maybe call it a "CSI volume clone" in this first instance for the module?

Great suggestion.